src/keyszer/config_api.py

```diff
@@ -150,33 +150,41 @@ class TypingTooLong(Exception):
 class UnicodeNumberToolarge(Exception):
     pass
 
+unicode_combo_list = []
```

Comment:
This global to communicate between the other two functions is a code smell/hack.

Reply:
Yes, but what to replace it with? Need a hint on the better way.

Reply:
Alright, never mind, I was being quite dense. The direct extend of combo_list negates the need for the global list entirely. 😮‍💨